README.md

```diff
@@ -4,7 +4,8 @@ CDN URL: https://npm-cdn.com/pkg/aria-query/
 
 # ARIA Query
 
-Programmatic access to the [WAI-ARIA 1.1 Roles Model](https://www.w3.org/TR/wai-aria-1.1/#roles).
+Programmatic access to the [WAI-ARIA 1.2 Roles Model](https://w3c.github.io/aria/#roles).
+This package tracks the editor's draft (last update: 25 June 2020).
```

Comment:
Not sure how they generate their spec. I visited it yesterday and the past 4 commits listed 25th June. Now they list 4th July. Seems like old specs aren't actually locked.